src/helpers/pressable-api-helper.php

```diff
@@ -58,14 +58,18 @@ public static function call_api( string $endpoint, string $method = 'GET', array
 		}
 		if ( 0 !== \strpos( $result['headers']['http_code'], '2' ) ) {
 			$message = '';
-			if ( \property_exists( $result['body'], 'message' ) ) {
-				$message = $result['body']->message;
-			} elseif ( \property_exists( $result['body'], 'error' ) ) {
-				$message = $result['body']->error;
-			}
-
-			if ( \property_exists( $result['body'], 'errors' ) && ! empty( $result['body']->errors ) ) {
-				$message .= ' ' . \implode( ', ', (array) $result['body']->errors );
+			if ( \is_null( $result['body'] ) ) {
+				$message = '{no response body 😭}';
+			} else {
+				if ( \property_exists( $result['body'], 'message' ) ) {
+					$message = $result['body']->message;
+				} elseif ( \property_exists( $result['body'], 'error' ) ) {
+					$message = $result['body']->error;
+				}
+
+				if ( \property_exists( $result['body'], 'errors' ) && ! empty( $result['body']->errors ) ) {
```

Comment:
Does this even need the `property_exists` check?  As I thought `empty` handled a `isset` internally.

Reply:
You are correct. The check for `! empty` is actually enough here -- I didn't realize that. However, all the `if`s in that part of the code make a check for `property_exists` so I think having it makes it ... uniform? We can safely remove it if you disagree though :)